govcd/access_control.go

```diff
@@ -68,7 +68,11 @@ func (client Client) GetAccessControl(href, entityType, entityName string, heade
 // * The subject (HREF and Type are mandatory)
 // * The access level (one of ReadOnly, Change, FullControl)
 func (client *Client) SetAccessControl(accessControl *types.ControlAccessParams, href, entityType, entityName string, headerValues map[string]string) error {
+	return client.setAccessControlWithMethod(http.MethodPost, accessControl, href, entityType, entityName, headerValues)
+}
 
+// setAccessControlWithMethod is the same as Client.SetAccessControl but allowing passing a different HTTP method
```

Comment:
Please also explain "the why": for what purposes one would want to use different HTTP methods here?

Reply:
Done!